### DISCUSSION TITLE
cache settings in mountainduck

### DISCUSSION THREAD

**Discussion by r91085** [iterate-ch/cyberduck#15382] [Q&A] [answered]
Is there any settings to extend the retention time of cache file more than 60 days ? I want to keep them forever until the size is full .
Thanks for help.

**Comment by dkocher** [comment]
Uncheck _Preferences… → Sync → Cache → Purge files…_ to retain cached files.

**Comment by r91085** [comment]
You mean if I don't enable it, then it would be kept forever ?

**Reply by dkocher** [reply]
That's correct.

**Comment by JosephMRally** [comment]
On a similar note. is there a way to force purge of the cache? Situation is i had it set to 1TB, lowered it and set it to 1 day, but i do not see it decreasing in the "group containers" on macos.

**Reply by tatze96** [reply, marked as answer]
The cache size limit applies to each bookmark individually. For example, if you have three bookmarks and use them within the *Smart Synchronization* mode, each bookmark can occupy the selected amount of cache in size. Files purposely marked to keep offline are excluded from the cache size limit.

The cache size can exceed the limit if you upload, open files exceeding the remaining cache space, or recently lowered the limit. Mountain Duck will clear out the oldest files from the cache once a day if the cache exceeds the set limit or the cached item exceeds the set cache expiry time (*Purge files in cache not opened after*). Mountain Duck needs to be running, and the corresponding bookmark must be connected to clear the cache.

You can free up some disk space manually by right-clicking on a file or folder and choosing `Mountain Duck → Delete on Local Disk` to clear out the file(s) from the cache. You can do the same while right-clicking on white space within the mount to clear out all files from the cache. Note that this will also clear out the files kept offline purposely.

Using the *Delete on Local Disk* option from the context menu won't prevent Mountain Duck from downloading the files to the cache again.

I generally don't recommend deleting the cache folder manually, as this can lead to file loss if there are synchronization errors, conflicts, or files that aren't fully synchronized yet.

For more information regarding the [cache limitation settings](https://docs.cyberduck.io/mountainduck/preferences/#cache-limitations), refer to the corresponding wiki article.
